docs/FAQ.md

```diff
@@ -19,6 +19,7 @@ The FAQ predated the introduction of the rest of the documentation. If something
 
 ## Behavior
 * [How does `dep` decide what version of a dependency to use?](#how-does-dep-decide-what-version-of-a-dependency-to-use)
+* [What is the default `dep ensure -update` behavior for dependencies that are imported but not included as a `[[Constraint]]` in `Gopkg.toml`?](#TODO-add-link)
```

Comment:
dont forget the link

Reply:
@arxdsilva I mentioned in the "Do you need help or clarification on anything" that I'm not sure what the link should be. I tried following the advice in [this SO post](https://stackoverflow.com/a/2823945), however Markdown wouldn't recognize the link it made. Do you have any suggestions?

Reply:
tried here inspecting the generated html and this tag worked: 

    #what-is-the-default-dep-ensure--update-behavior-for-dependencies-that-are-imported-but-not-included-as-a-constraint-in-gopkgtoml

Reply:
:bump:

Reply:
when i'm not sure, i usually go to just preview the file in the PR diff in normal view/not raw mode, then see what anchor link it generated 😄 in this case:

`what-is-the-default-dep-ensure--update-behavior-for-dependencies-that-are-imported-but-not-included-as-a-constraint-in-gopkgtoml`

Reply:
@sdboyer Thank you! Fixing now and rebasing, should be ready in 15m or so.

Reply:
@sdboyer Updated!